libc/include/limits.h

```diff
@@ -0,0 +1,16 @@
+#ifndef __LIMITS_H
+#define __LIMITS_H
+
+/* Maximum number of bytes in a pathname, including the terminating null. */
+#define PATH_MAX 128
+
+/* Maximum number of bytes in a filename, not including terminating null. */
+#define NAME_MAX 14
```

Comment:
We're going to need NAME_MAX to be 26 for the FAT filesystem. See linuxmt/dirent.h. `(readdir` returns from this structure). ELKS' implementation of the FAT filesystem allows for VFAT (long filename) entries with up to two dir entries (13 bytes each), which ends up allowing for 26 bytes total for FAT (and 14 for MINIX).